### ISSUE TITLE
Licence clashes with README EULA

### ISSUE THREAD

**Issue by WindowxDeveloper** [zeropolis79/PETSCIIRobots-C64REU#1]
Hi, there, Zeropolis.

I just noticed that your work is GPL'd. I don't want to be "that guy", but the problem is your README file explicitly stated that most modification and commercial selling is forbidden, which would clash with the GNU GPL (that _does_ allow the aforementioned acts to be done, as long as the corresponding source code remains available). May you take your time to choose a different licence or revamp the README?

Thank you for your attention.

**Comment by metalx1000** [comment]
@WindowxDeveloper I was going to submit the same thing. 

The GPLv3 is a great license, but you can't license it as that and then put contradicting restrictions on it.  The GPL is a distribution license, not a EULA.  Licensing it with a GPL means that the user can use it for whatever they see fit and the GPL is suppose to make sure they can share it, both for commercial use and non-commercial use.

**Comment by zeropolis79** [comment]
I'm sorting that problem out

**Comment by InklingGirl** [comment]
> I'm sorting that problem out

Yeah, 'sorting that problem out' … by just pretending you didn't use the license & disrespecting the end-user's freedom/liberty by imposing restrictions on how we can use the code. Not only are @zeropolis79  & David in breach of literally any version of the GNU GPL, but David can't sit there in his videos & claim his games are [free/libre](https://www.gnu.org/philosophy/free-sw.en.html) & '[open-source](https://opensource.org/osd/)' (the term [he uses](https://youtu.be/3kyf3aOIu_U?si=kn-JUIBPubEdyhVa&t=1444)) software as per the actual definitions of these terms.
